lib_test/test_icmpv4.ml

```diff
@@ -78,6 +82,15 @@ type stack = {
   icmp : Icmp.t;
 }
 
+let testbind x y =
+  match x with
+  | Result.Ok p -> y p
+  | Result.Error s -> Alcotest.fail s
+let (>>=?) = testbind
+
+let slowly fn =
+  OS.Time.sleep 0.1 >>= fun () -> fn >>= fun () -> OS.Time.sleep 0.1
```

Comment:
Note that `fn` starts before `slowly` is called. This function waits 0.1s, then waits for `fn` to complete, then waits another 0.1s. Probably it should be `fn ()`. Also, possibly it should `Lwt_unix.yield` instead of sleeping? What is it for?